platform/viewer/src/appExtensions/MeasurementsPanel/ConnectedMeasurementTable.js

```diff
@@ -58,6 +59,19 @@ function getDataForEachMeasurementNumber(
   return data;
 }
 
+function getWarningsForMeasurement(meansurementData) {
+  const { toolType } = meansurementData;
```

Comment:
Should `meansurementData` be `measurementData`?
Also, why pass in the entire object if we only use `toolType`?
Is `toolType` a string? Can we add JSDoc?

Reply:
Fixed the callouts. 
I think my idea was to get warnings for measurements not only related with the toolType, but end up only using the toolType. Just removed and fixed the callout.